test/classes/corrado/type_alias_bug.verify.future

```diff
@@ -0,0 +1 @@
+cannot catch a type alias of a class with --verify: https://github.com/chapel-lang/chapel/issues/22720
```

Comment:
Please switch to an appropriate format of a .future file, which is:
```
bug: cannot catch a type alias of a class
#22720
```

I suggest that "cannot catch" is not limited to compilations with --verify, given that without --verify we still get an asan failure.